### ISSUE TITLE
pip install fails without jupyterlab

### ISSUE THREAD

**Issue by martinal** [jovyan/pythreejs#102]
Not sure it's exactly because jupyterlab is not installed, but I get this error on `pip install .`:
```
   ...
    creating /home/martinal/envs/env-versioned/share/jupyter/labextensions
    creating /home/martinal/envs/env-versioned/share/jupyter/labextensions/jupyter-threejs
    error: can't copy 'pythreejs/staticlab/jupyter-threejs.bundle.js': doesn't exist or not a regular file
```

**Comment by jzf2101** [comment]
@jasongrout We discussed seeing if we could get pythreejs ready for jupyterlab beta and the jupyterlab demo.  Any news on this?

**Comment by vidartf** [comment]
Fixed in 0.4.x and master.
